### ISSUE TITLE
Kit name and variant short name accessable by variable substitution 

### ISSUE THREAD

**Issue by KoeMai** [vector-of-bool/vscode-cmake-tools#286]
### Brief Issue Summary

I use for development different compilers with the same root CmakeList.txt File. It would be helpful to have a separate build directory for each compiler + variant.  This speed up the switch between the variants or compiler build environments, because no complete CMake regeneration is needed, only a configure.

### Expected:

The build directory path is changable by the settings. I would like to Set into the build directory path over variable substitution the following information:

- the kit name
- the short of the variants

### Platform and Versions

- **Operating System**: Windows 10
- **CMake Version**: 3.9
- **VSCode Version**: 1.19
- **CMake Tools Extension Version**: Plan to us it
- **Compiler/Toolchain**:  GCC 6.3 Windows and GCC for ARM

**Comment by hermanzdosilovic** [comment]
I am in the same situation.

This is what I currently have in my workspace `settings.json`:
```
"cmake.buildDirectory": "${userHome}/Builds/${workspaceRootFolderName}/${buildType}"
```

and this is what would be very nice to have:
```
"cmake.buildDirectory": "${userHome}/Builds/${workspaceRootFolderName}/${buildType}/${kitName}"
```

Suggesting `kitName` as a variable that would be available for us to use in `settings.json`. :smiley:

**Comment by Randshot** [comment]
Fixed in #317.
**Closed**.
If this issues still persists in the current **_0.11.0-beta4_**, feel free to reopen this one or create a new one.

**Comment by Randshot** [comment]
PR #317 merged.
